app/services/decorators/v6/applicant_disposable_income_result_decorator.rb

```diff
@@ -21,7 +22,10 @@ def as_json
     private
 
       def proceeding_types
```

Comment:
@starswan What do you think of using some base result decorator to remove the duplication?

Reply:
doesn't fit well with the LSP (Liskov Substitution Principal). Think maybe an eligibility should keep a reference to its own proceeding type though...